src/net/sourceforge/kolmafia/persistence/ItemFinder.java

```diff
@@ -39,15 +39,15 @@ public enum Match {
     ASDON,
   }
 
-  public static final List<String> getMatchingNames(String searchString) {
+  public static List<String> getMatchingNames(String searchString) {
     return ItemDatabase.getMatchingNames(searchString);
   }
 
-  public static final String getFirstMatchingItemName(List<String> nameList, String searchString) {
+  public static String getFirstMatchingItemName(List<String> nameList, String searchString) {
     return ItemFinder.getFirstMatchingItemName(nameList, searchString, Match.ANY);
   }
 
-  public static final String getFirstMatchingItemName(
+  public static String getFirstMatchingItemName(
       List<String> nameList, String searchString, Match filterType) {
     if (nameList == null || nameList.isEmpty()) {
```

Comment:
I have a failing test and I don't understand whether it is my misunderstanding or whether the underlying code is wrong.
The parameter searchString is not used which seems to me to be an error.